prac_05/word_occurrences.py

```diff
@@ -6,6 +6,19 @@
 user_text = input(str("Please enter a string of text: "))
 
 words = user_text.split()
+#Counting the words from the users input
+for word in words:
+    word_occurrence = users_words.get(word, 0)
+    if word_occurrence is None:
+        users_words[word] = 1
+    else:
+        users_words[word] = word_occurrence + 1
+
+words = list(users_words.keys())
```

Comment:
Did you think about making lines 17-19 in to a function?

Reply:
Hello Sarah, 
No I didn't, I will have a play with my code and see if I can make a function in that program. 
Thank you for your input.